tensorflow/python/keras/saving/saving_utils.py

```diff
@@ -219,6 +219,18 @@ def should_overwrite(filepath, overwrite):
   return True
 
 
+def convert_output_metrics(metrics_config, custom_objects):
+  from google3.third_party.tensorflow.python.keras import metrics as metrics_module  # pylint:disable=g-import-not-at-top
```

Comment:
This can be removed.
from tensorflow.python.keras import metrics as metrics_module  # pylint:disable=g-import-not-at-top